src/debug/Stream.h

```diff
@@ -37,10 +37,11 @@
 #define MAX_DEBUG_SECTIONS 100
 
 /* defined names for Debug Levels */
-#define DBG_CRITICAL    0   /**< critical messages always shown when they occur */
-#define DBG_IMPORTANT   1   /**< important messages always shown when their section is being checked */
-/* levels 2-8 are still being discussed amongst the developers */
-#define DBG_DATA    9   /**< output is a large data dump only necessary for advanced debugging */
+#define DBG_CRITICAL   0 /**< critical messages always shown when they occur */
+#define DBG_IMPORTANT  1 /**< important messages always shown when their section is being checked */
+#define DBG_PROTOCOL   2 /**< show flow of protocol messages. Excluding message content/payload which is in DBG_DATA. */
+/* levels 3-8 are still being discussed amongst the developers */
+#define DBG_DATA       9 /**< output is a large data dump only necessary for advanced debugging */
 
```

Comment:
Please avoid modifying otherwise unchanged lines above just to align their number "column". We should avoid such decorations because they force us to modify code that should not be touched without adding sufficient value in return.

Reply:
Define "sufficient" ?

Reply:
Whether a given value is sufficient to overcome known problems cannot be defined -- it is a judgement call. The author is responsible for convincing the reviewer that reformatting three out of three old definitions to gain alignment is worth causing all those known problems, including the time spent on this change request. So far, I am unconvinced. My recommendation is to cut our losses by keeping the official definition lines unchanged (and to avoid creating similar problems in the future by avoiding touching the lines to align something that was not aligned and does not need to be aligned).

Reply:
Ergo, you are imposing on all contributors a requirement that every change no matter how small be justified and defended extensively to some undefined arbitrary standard of "quality". Any contributor attempting to improve code readability that does not go to those unreasonable length in advance is forced to expend extra work and time either reverting trivial formatting _improvements_ of justifying some ROV retrospectively.

IMO that is unreasonable expectation. Contributors should have the right to choose how they contribute without having to justify self-evident code polish.

Reply:
> you are imposing on all contributors a requirement that every change no matter how small be justified and defended extensively to some undefined arbitrary standard of "quality".

I do not. Process improvement suggestions are welcome.

> Contributors should have the right to choose how they contribute without having to justify self-evident code polish.

I agree. In this specific case, we probably disagree on a few _other_ things, including whether introducing or fixing an unwelcomed alignment is "self-evident code polish" or an easily fixable mistake.

Reply:
Dropped with DBG_PROTOCOL removal in [e3a7b48](https://github.com/squid-cache/squid/pull/1105/commits/e3a7b48eb0e00683d6b7284126ccbf260c6ec64b)